lib/engine/game/g_1880/step/assign.rb

```diff
@@ -74,6 +75,8 @@ def process_assign_rocket(action)
             @game.rocket.close!
             @game.buy_train(buying_corp, train, :free)
             @game.phase.buying_train!(buying_corp, train)
+            @game.depot.export_all!(train.name, silent: true) if @round.special_edge_case_4t_rocket
+            @round.special_edge_case_4t_rocket = false
```

Comment:
It seems like if `special_edge_case_4t_rocket` is already `false`, so you don't need to re-set it every time? So maybe this works:

```suggestion
            if @round.special_edge_case_4t_rocket
              @game.depot.export_all!(train.name, silent: true) 
              @round.special_edge_case_4t_rocket = false
            end
```